### PR TITLE
Implement products sort, and Integrate Standalone Mode to save up ~9MB

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
The change results in a much faster app: https://rishi-raj-jain-layer0-nextjs-example-default.layer0-limelight.link

The app size when switching to the standalone mode went from 11.X MB to 2.X MB, which is a great win.

### CODE REVIEW

File: components/LeftSidebar.js [hunk 1]
```diff
@@ -5,26 +5,20 @@ import classNames from 'classnames'
 const listingItems = {
   'All Categories': [
     {
-      name: 'Apparel',
-      route: '/commerce/apparel',
+      name: 'Joggers',
+      route: '/commerce/joggers',
     },
     {
-      name: 'Shop All',
```
Comment (jeffpatzer) [inline components/LeftSidebar.js:12]: Why fully pull the "Shop all"?

File: pages/_app.js [hunk 2]
```diff
@@ -1,27 +1,30 @@
-import Layer0RUM from 'rum'
 import '@/styles/globals.css'
 import { useEffect } from 'react'
+import Layer0RUM from '@/layer0/rum'
```
Comment (jeffpatzer) [inline pages/_app.js:3]: typo? not `@layer0/rum`?

File: package.json [hunk 3]
```diff
@@ -10,22 +10,22 @@
...
-    "@layer0/prefetch": "^4.15.6",
-    "@layer0/react": "^4.15.6",
+    "@heroicons/react": "^1.0.6",
+    "@layer0/cli": "^4.15.7",
+    "@layer0/core": "^4.15.7",
+    "@layer0/devtools": "^4.15.7",
+    "@layer0/next": "^4.15.7",
+    "@layer0/prefetch": "^4.15.7",
+    "@layer0/react": "^4.15.7",
+    "@layer0/rum": "^3.1.0",
```
Comment (jeffpatzer) [inline package.json:25]: This is used in `_app.js` shouldn't it be a full dependency?

File: pages/product/[name].js [hunk 4]
```diff
@@ -30,7 +29,7 @@ const Product = ({ data }) => {
           </div>
           <HeartIcon className="absolute top-0 right-0 h-[50px] w-[50px] bg-white p-2" />
           <div className="relative flex h-[600px] w-full flex-col items-center">
-            <NextImage layout="fill" objectFit="contain" src={data.images[selectedImage].url} className="h-auto w-full max-w-[600px]" />
+            <img src={relativizeURL(data.images[selectedImage].url)} className="h-auto w-full max-w-[600px] object-contain" />
```
Comment (jeffpatzer) [inline pages/product/[name].js:32]: IMages are brutally slow. Can we add Image Optimization?

File: next.config.js [hunk 5]
```diff
@@ -1,40 +1,3 @@
...
-  // For image optimization using Next.js Image's component
-  // Read more on: https://nextjs.org/docs/basic-features/image-optimization
-  images: {
-    domains: ['layer0-docs-layer0-ecommmerce-api-example-default.layer0-limelight.link'],
-  },
+module.exports = {
+  output: 'standalone',
 }
-
-module.exports = (phase, config) =>
```
Comment (jeffpatzer) [inline next.config.js:26]: How does removing this in the config affect the service worker?

**Review by jeffpatzer — CHANGES_REQUESTED** [review]
Nice updates. A few questions and comments in the PR.

File: components/LeftSidebar.js (hunk 1 above)
Comment (rishi-raj-jain) [reply components/LeftSidebar.js:12]: If you load: https://rishi-raj-jain-layer0-nextjs-example-default.layer0-limelight.link/commerce, then Shop All is still there. It's taken away from that particular line but included inside the sidebar.

File: pages/_app.js (hunk 2 above)
Comment (rishi-raj-jain) [reply pages/_app.js:3]: Nope, all the Layer0 specific files have been moved into a layer0 folder. This folder as mentioned in jsconfig,json can be accessed directly via @/layer0 prefix.

File: package.json (hunk 3 above)
Comment (rishi-raj-jain) [reply package.json:25]: Nope, that is compiled during the build time.

Even if that required that library, layer0/build.js would smartly include all dependencies that would be required in runtime.

File: pages/product/[name].js (hunk 4 above)
Comment (rishi-raj-jain) [reply pages/product/[name].js:32]: Oh yes, Added in https://github.com/layer0-docs/layer0-nextjs-example/pull/117/commits/b0ba9e472c778434b501d7153299b55ccd7b485b

File: next.config.js (hunk 5 above)
Comment (rishi-raj-jain) [reply next.config.js:26]: It doesn't affect the service worker at all. We take the process done by @layer0/next that currently doesn't support standalone mode to ours. layer0/build.js handles this all. That config by Layer0 is just used to inject service worker, which is being handled already.

**Review by jeffpatzer — APPROVED** [review]

**Pull request merged**